### DISCUSSION TITLE
Regarding "Reset" and "Factoryreset" commands

### DISCUSSION THREAD

**Discussion by hastigondaliya** [openthread/openthread#9404] [Q&A] [answered]
As Openthread CLI supports **reset** and **factoryreset**. 

We've some queries regarding these commands mentioned below:

1. As per README, the factoryreset command deletes all stored settings, and signal a platform reset.
 Could you please clarify which settings are mandatory to be deleted and which are specific to the manufacturer (like Tx Power, RSSI)?
2. What is distinction between reset and factoryreset commands?

**Comment by jwhui** [comment, marked as answer]
> As per README, the factoryreset command deletes all stored settings, and signal a platform reset.
Could you please clarify which settings are mandatory to be deleted and which are specific to the manufacturer (like Tx Power, RSSI)?

A factory reset must clear all non-volatile settings defined in [include/openthread/platform/settings.h](https://github.com/openthread/openthread/blob/main/include/openthread/platform/settings.h).

> What is distinction between reset and factoryreset commands?

Reset simply restarts the Thread stack and does not clear non-volatile memory.

Factory restarts the Thread stack and clears non-volatile memory.

**Comment by hastigondaliya** [comment]
Hi @jwhui 

Thank you for the clarification.
